students/TianYen/Session02/FizzBuzz.py

```diff
@@ -0,0 +1,17 @@
+def FizzBuzz(n):
+    """print the first n numbers, if n is a multiple of 3 it will print Fizz,
+        if n is a multiple of 5 it will print buzz, and if it is a multiple of both it
+        will print FizzBuzz"""
+    for i in range(1, n + 1):
+        word = ""
+        if (i) % 3 == 0:
+            word += "Fizz"
+        if (i) % 5 == 0:
+            word += "Buzz"
+        if len(word) > 0:
```

Comment:
This is in this week's reading: https://uwpce-pythoncert.github.io/PythonCertDevel/modules/Booleans.html

but better to simply write:

```
if word:
    ...
```

a string will return false if it is empty.